docker/Dockerfile

```diff
@@ -65,10 +65,6 @@ RUN mkdir /usr/local/share/java
 
 WORKDIR /home/Downloads/
 
-# JWNL - Never changes, so do this first.
-ADD http://downloads.sourceforge.net/project/jwordnet/jwnl/JWNL%201.4/jwnl14-rc2.zip /home/Downloads/jwnl14-rc2.zip
-RUN (unzip jwnl14-rc2.zip; cd jwnl14-rc2; cp jwnl.jar /usr/local/share/java/; chmod 755 /usr/local/share/java/jwnl.jar)
-
 # OpenNLP - Never changes, so do this first.
```

Comment:
Sorry for not catching at first time, but it is not needed as well.

Reply:
Updated.